iservan-cadiz.md

```diff
@@ -0,0 +1,12 @@
+# Cádiz, Andalucia, España
+
+### Restaurants
+
+### Atractions for tourist
+
+- Go to the beach and coastine
+- Visit the old center twon & the cathedral
+
+### Atractions for locals
+
+- Go to the beach
```

Comment:
@xiscapericas if you travel around Cadiz you can't miss **Playa de los Alemanes** in Zahara de los Atunes 🐟 🐠 🌊 ☀️ 😃